python-test/features/steps/agentsProvider.py

```diff
@@ -0,0 +1,55 @@
+from behave import given, when, then
+import time
+import functions
+
+base_orb_url = "https://beta.orb.live"
+email = 'tester@email.com'
+password = '12345678'
+
+agentname = 'agent1'
+agentTagKey = 'test'
+agentTagValeu = 'true'
+
+
+@given('A valid authentication')
+def get_auth(context):
+    context.token = functions.generate_token(email, password, base_orb_url)
+    assert len(context.token) > 0
+
+@when('Create an agent')
+def create_agent(context):
+    context.status_agent, context.agent_data = functions.new_agent(context.token, agentname, agentTagKey, agentTagValeu)
+    assert context.status_agent == 201
+    context.agent_key = context.agent_data['key']
+    context.agent_id = context.agent_data['id']
+    context.agent_channel_id = context.agent_data['channel_id']
+
+@when('Build agent container')
```

Comment:
this should say "Run agent container", because "building" a container is a different operation

Reply:
Thanks. I'm gonna change it